pyproject.toml

```diff
@@ -109,31 +118,31 @@ known-local-folder = ["tuning"]
 [tool.mypy]
 disable_error_code = ["import-not-found", "import-untyped"]
 exclude = [
-    "^src/instructlab/model/chat\\.py$",
-    "^src/instructlab/model/convert\\.py$",
-    "^src/instructlab/model/download\\.py$",
-    "^src/instructlab/model/test\\.py$",
-    "^src/instructlab/model/serve\\.py$",
-    "^src/instructlab/model/list\\.py$",
-    "^src/instructlab/model/model\\.py$",
-    "^src/instructlab/model/train\\.py$",
-    "^src/instructlab/config/config\\.py$",
-    "^src/instructlab/config/init\\.py$",
-    "^src/instructlab/data/data\\.py$",
-    "^src/instructlab/data/generate\\.py$",
-    "^src/instructlab/data/generator/generate_data\\.py$",
-    "^src/instructlab/data/generator/utils\\.py$",
-    "^src/instructlab/taxonomy/taxonomy\\.py$",
-    "^src/instructlab/taxonomy/diff\\.py$",
-    "^src/instructlab/llamacpp/llamacpp_convert_to_gguf\\.py$",
-    "^src/instructlab/mlx_explore/gguf_convert_to_mlx\\.py$",
-    "^src/instructlab/mlx_explore/utils\\.py$",
-    "^src/instructlab/train/lora_mlx/lora\\.py$",
-    "^src/instructlab/train/lora_mlx/models/llama\\.py$",
-    "^src/instructlab/train/lora_mlx/models/mixtral\\.py$",
-    "^src/instructlab/train/lora_mlx/models/models\\.py$",
-    "^src/instructlab/utils\\.py$",
-    "^src/instructlab/server\\.py$",
+  "^src/instructlab/model/chat\\.py$",
+  "^src/instructlab/model/convert\\.py$",
+  "^src/instructlab/model/download\\.py$",
+  "^src/instructlab/model/test\\.py$",
+  "^src/instructlab/model/serve\\.py$",
+  "^src/instructlab/model/list\\.py$",
+  "^src/instructlab/model/model\\.py$",
+  "^src/instructlab/model/train\\.py$",
+  "^src/instructlab/configuration/config\\.py$",
```

Comment:
I think this is wrong; the files were renamed in tree.

Reply:
You're right, messed up my rebase it seems :) Good catch!